tests/test_edges.py

```diff
@@ -805,13 +754,23 @@ def test_basic(self):
                 test_module.DYNAMICS_PREFIX + "param1",
             ]
         )
-        assert self.test_obj.type == DEFAULT_EDGE_TYPE
+        # TODO: change to DEFAULT_EDGE_TYPE when fixed in libsonata (= "chemical")
```

Comment:
is this fixed?

Reply:
It seems this is `chemical_synapse` in `libsonata`: https://github.com/BlueBrain/libsonata/blob/master/src/config.cpp#L744

Reply:
huh; the doc says it's also `chemical`: https://sonata-extension.readthedocs.io/en/latest/sonata_config.html#populations

but I swear it's inconsistent somewhere else - let me look